stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/IOConstants.java

```diff
@@ -42,4 +42,6 @@
      * The size of the buffer allocated for reading bytes from the channel (15~ KB).
      */
     public static final int CHANNEL_BUFFER_SIZE = 16384;
+
+    public static final String CLIENT_SOCKET_NAME = "clientSocket";
```

Comment:
If possible, Shall we document this constant as other cases? Seems it is referred as key for native data for struct.